kroto-plus-coroutines/src/test/kotlin/com/github/marcoferrer/krotoplus/coroutines/server/ServerCallBidiStreamingTests.kt

```diff
@@ -128,10 +133,11 @@ class ServerCallBidiStreamingTests {
                 respChannel = responseChannel
                 var groupCount = 0
                 val requestIter = requestChannel.iterator()
+
                 while (requestIter.hasNext()) {
                     val requestValues = listOf(
-                        requestIter.next(),
-                        requestIter.next(),
+                        requestIter.next().also { requestIter.hasNext() },
+                        requestIter.next().also { requestIter.hasNext() },
```

Comment:
This was required after updating to coroutines `1.3.0-RC`. Every call to `next()` must be preceded by a call to `hasNext()`